twentytwenty-blocks/functions.php

```diff
@@ -0,0 +1,309 @@
+
+		// Add support for full and wide align images.
+		add_theme_support( 'align-wide' );
+
+		// Add support for editor styles.
+		add_theme_support( 'editor-styles' );
+
+		// Enqueue editor styles.
+		add_editor_style( 'twentytwenty-styles/editor-style-block.css' );
+
+		// Add custom editor font sizes.
+		add_theme_support(
+			'editor-font-sizes',
+			array(
+				array(
+					'name'      => __( 'Small', 'twentytwenty-blocks' ),
+					'shortName' => __( 'S', 'twentytwenty' ),
+					'size'      => 19.5,
+					'slug'      => 'small',
+				),
+				array(
+					'name'      => __( 'Normal', 'twentytwenty-blocks' ),
+					'shortName' => __( 'M', 'twentytwenty' ),
+					'size'      => 22,
+					'slug'      => 'normal',
+				),
+				array(
+					'name'      => __( 'Large', 'twentytwenty-blocks' ),
+					'shortName' => __( 'L', 'twentytwenty' ),
+					'size'      => 36.5,
+					'slug'      => 'large',
+				),
+				array(
+					'name'      => __( 'Huge', 'twentytwenty-blocks' ),
+					'shortName' => __( 'XL', 'twentytwenty' ),
+					'size'      => 49.5,
+					'slug'      => 'huge',
+				),
+			)
+		);
+
+		// Editor color palette.
+		add_theme_support(
+			'editor-color-palette',
+			array(
+				array(
+					'name'  => __( 'Primary', 'twentytwenty-blocks' ),
+					'slug'  => 'primary',
+					'color' => '#0073AA',
+				),
+				array(
+					'name'  => __( 'Secondary', 'twentytwenty-blocks' ),
+					'slug'  => 'secondary',
+					'color' => '#005177',
+				),
+				array(
+					'name'  => __( 'Dark Gray', 'twentytwenty-blocks' ),
+					'slug'  => 'dark-gray',
+					'color' => '#111',
+				),
+				array(
+					'name'  => __( 'Light Gray', 'twentytwenty-blocks' ),
+					'slug'  => 'light-gray',
+					'color' => '#767676',
+				),
+				array(
+					'name'  => __( 'White', 'twentytwenty-blocks' ),
+					'slug'  => 'white',
+					'color' => '#FFF',
+				),
+			)
+		);
+
+		// Add support for responsive embedded content.
+		add_theme_support( 'responsive-embeds' );
+
+		// Starter content
+		add_theme_support('starter-content', [
+			// Static front page set to Front
+			'options' => [
+				'show_on_front' => 'page',
+				'page_on_front' => '{{front}}',
+			],
+
+			// Create the custom image attachments used as post thumbnails for pages.
+			'attachments' => array(
+				'image-opening' => array(
+					'post_title' => _x( 'The New UMoMA Opens its Doors', 'Theme starter content', 'twentytwenty-blocks' ),
+					'file'       => 'assets/images/2020-landscape-1.png', // URL relative to the template directory.
+				),
+			),
+
+			// Specify the core-defined pages to create and add custom thumbnails to some of them.
+			'posts' => array(
+				'front' => array(
+					'post_type'    => 'page',
+					'post_title'   => __( 'The New UMoMA Opens its Doors', 'twentytwenty-blocks' ),
+					// Use the above featured image with the predefined about page.
+					'thumbnail'    => '{{image-opening}}',
+					'post_content' => join(
+						'',
+						array(
```

Comment:
I'm a little torn about how to handle this starter content still: 

For our purposes now, I do think it would help code readability to move this into template part, but I'm not sure it actually makes sense to at this point — since this is just standard page content, users will want to edit this inside the standard page editor, not in the block templates area of the editor.

Reply:
I think what we're really looking to use is "Page Templates", as per https://github.com/WordPress/gutenberg/issues/18055. That functionality doesn't appear to be ready for themes too hook into yet, so sticking with the starter content is probably the best option we have at the moment. I've left some feedback in that ticket here: https://github.com/WordPress/gutenberg/issues/18055#issuecomment-576336048.

Reply:
While it does improve readability at this moment, I agree that putting starter content in a block template is not the right approach.